src/cmds/qstat.c

```diff
@@ -302,47 +361,163 @@ states(char *string, char *q, char *r, char *h, char *w, char *t, char *e, int l
  * @param[in] n - attribute name
  * @param[in] r - resource name
  * @param[in] v - value for the attribute
+ * @param[in] one_line - one line per attribute
  *
  * @return Void
  *
  */
 void
-prt_attr(char *n, char *r, char *v)
-{
-	char *c;
+prt_attr(char *name, char *resource, char *value, int one_line) {
```

Comment:
Thank you for changing the variable names - same needs to reflect in the doxygen above..